azure-pipelines.yml

```diff
@@ -172,7 +172,7 @@ jobs:
   - bash:  ci/scripts/check_bazel_target_names.py
     displayName: Check Bazel target names (Experimental)
     continueOnError: True
-  - bash: ci/scripts/check-generated.sh
+  - bash: OT_DESTRUCTIVE=1 ci/scripts/check-generated.sh
```

Comment:
Could you add a quick comment above this line explaining this variable for other people working with this file?